dockerfiles/che-fabric8/Dockerfile

```diff
@@ -4,9 +4,11 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 ARG CHE_DASHBOARD_VERSION=next
+ARG CHE_DASHBOARD_NEXT_VERSION=next
 ARG CHE_WORKSPACE_LOADER_VERSION=next
 
 FROM quay.io/eclipse/che-dashboard:${CHE_DASHBOARD_VERSION} as che_dashboard_base
+FROM quay.io/che-incubator/che-dashboard-next:${CHE_DASHBOARD_VERSION} as che_dashboard_next_base
```

Comment:
@sleshchenko should be `CHE_DASHBOARD_NEXT_VERSION`

Reply:
good catch! Fixed.